kymatio/__init__.py

```diff
@@ -13,7 +16,14 @@
             ]
 
 from .scattering1d.scattering1d import Scattering1D
-from .scattering2d.scattering2d import Scattering2D
+
+# if torch is loaded
+if 'torch' in sys.modules:
```

Comment:
Can you explain what you intend here?

This looks to me like the user has to import torch? And then this gets detected by kymatio?

If this is the case, then I don't think it's a good idea.

Reply:
Okay. I agree, I will remove this.